src/index.mako.html

```diff
@@ -233,6 +230,14 @@
     </div>
 % endif
 
+% if device == 'mobile':
+    <div ga-background-selector ga-background-selector-map="map" class="ga-bg-mobile" ng-show="!globals.offline"></div>
+% endif
+
+% if device == 'embed':
+    <div ga-background-selector ga-background-selector-map="map"></div>
```

Comment:
I removed it.

2015-06-09 12:50 GMT+02:00 oterral <notifications@github.com>:

> In src/index.mako.html
> <https://github.com/geoadmin/mf-geoadmin3/pull/2397#discussion_r32001193>:
>
> > @@ -233,6 +230,14 @@
> >      </div>
> >  % endif
> >
> > +% if device == 'mobile':
> > +    <div ga-background-selector ga-background-selector-map="map" class="ga-bg-mobile" ng-show="!globals.offline"></div>
> > +% endif
> > +
> > +% if device == 'embed':
> > +    <div ga-background-selector ga-background-selector-map="map"></div>
>
> Where is the bg selector on embed page ? I don't see it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoadmin/mf-geoadmin3/pull/2397/files#r32001193>.
>



-- 
Twitter: http://twitter.com/cedricmoullet
Linked In: http://www.linkedin.com/in/cedricmoullet

<http://twitter.com/cedricmoullet>